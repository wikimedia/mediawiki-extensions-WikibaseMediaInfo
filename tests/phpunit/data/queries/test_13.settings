{
	"title": "No weighted tags or statements",
	"term": "some search term",
	"entitiesForTerm": {
		"some search term": [
			{
				"id": "Q1",
				"score": 1.0,
				"synonyms": [
					"Some thing",
					"Something"
				]
			},
			{
				"id": "Q2",
				"synonyms": [
					"Another thing"
				]
			}
		]
	},
	"defaultProperties": [
		"P1",
		"P2"
	],
	"userLanguage": "en",
	"settings": {
		"normalizeFulltextScores": false,
		"normalizeMultiClauseScores": false,
		"entitiesVariableBoost": true,
		"applyLogisticFunction": false,
		"useSynonyms": false,
		"boost": {
			"statement": 0,
			"descriptions.$language": 0,
			"descriptions.$language.plain": 0,
			"category": 0,
			"category.plain": 0,
			"title": 9,
			"title.plain": 0,
			"redirect.title": 0,
			"redirect.title.plain": 0,
			"text": 0,
			"text.plain": 0,
			"auxiliary_text": 0,
			"auxiliary_text.plain": 0,
			"file_text": 0,
			"file_text.plain": 0,
			"heading.plain": 0,
			"heading": 0,
			"suggest": 0
		}
	}
}
