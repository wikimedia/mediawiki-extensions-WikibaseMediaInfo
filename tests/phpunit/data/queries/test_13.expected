{
    "bool": {
        "must": [
            {
                "bool": {
                    "minimum_should_match": 1,
                    "should": [
                        {
                            "dis_max": {
                                "tie_breaker": 1,
                                "queries": [
                                    {
                                        "bool": {
                                            "minimum_should_match": 0,
                                            "filter": [
                                                {
                                                    "multi_match": {
                                                        "query": "some search term",
                                                        "fields": [
                                                            "all",
                                                            "all.plain"
                                                        ],
                                                        "operator": "and"
                                                    }
                                                }
                                            ],
                                            "should": [
                                                {
                                                    "match": {
                                                        "title": {
                                                            "query": "some search term",
                                                            "boost": 9
                                                        }
                                                    }
                                                }
                                            ]
                                        }
                                    }
                                ]
                            }
                        },
                        {
                            "match_none": {}
                        }
                    ]
                }
            }
        ],
        "filter": [
            {
                "terms": {
                    "namespace": [
                        0
                    ]
                }
            }
        ]
    }
}